lib/internal/Magento/Framework/Exception/FeedImporterException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Feed importer exception
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\Exception;
```

Comment:
If you look at the Zend_Feed module Zend_Feed_Exception can be thrown for a lot of things. The only common exception for all those cases would be RuntimeException, so I picked that one. Let me know what you think.